tests/geometry/WorkingGeometry.spec.ts

```diff
@@ -60,16 +56,18 @@ describe('WorkingGeometry', () => {
                 vec3.fromValues(1, 1, 0),
                 vec3.fromValues(1, 0, 0)
             ];
-            const normal = vec3.fromValues(0, 0, 1);
-            const faces = [new Face([0, 1, 2], normal), new Face([0, 2, 3], normal)];
+            const faces = [new Face([0, 1, 2]), new Face([0, 2, 3])];
             const controlPoints = [vec3.fromValues(0, 0, 0)];
             const square = new WorkingGeometry(vertices, faces, controlPoints);
             const bakedSquare = square.bake();
 
             // Not testing the colors yet since they don't do anything useful
             expect(bakedSquare.indices).toEqual([0, 1, 2, 0, 2, 3]);
             expect(bakedSquare.vertices).toEqual(vertices);
-            expect(bakedSquare.normals).toEqual([normal, normal]);
+            expect(bakedSquare.normals).toEqual([
+                vec3.fromValues(-0, -0, -1),
```

Comment:
nit: do we really need -0's? I think 0 would work, right?

Reply:
Currently vec4.cross returns -0's... which doesn't seem to equal to 0 in the tests. Will look into it for now and see if there's anything I can do to fix this. I agree that it would be a lot nicer w/o the -0s, but I don't think it should particularly block the PR.